src/views/getProfile.js

```diff
@@ -12,8 +12,8 @@ export class Profile extends Component {
   }
 
   render() {
-    const { fetched } = this.props;
-    if (!fetched) {
+    const { isFetching } = this.props;
+    if (!isFetching) {
```

Comment:
Here, it should be isFetching, that is when fetching data while making a request.

Reply:
Have you checked on this?